fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/connection/SetSchemaAction.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.connection;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.ibm.fhir.persistence.jdbc.exception.FHIRPersistenceDBConnectException;
+
+
+/**
+ * Command to set the named schema on a connection
+ */
+public class SetSchemaAction extends ChainedAction {
+    private static final Logger log = Logger.getLogger(SetSchemaAction.class.getName());
+
+    // supplier to obtain the schema name when we need it
+    private final SchemaNameSupplier schemaNameSupplier;
+    
+    /**
+     * Use a provided schema name (handy for testing)
+     * @param schemaName
```

Comment:
You are missing the right names

Reply:
?

Reply:
Does this comment refer to javadoc? It should be correct.